source/server/admin/logs_handler.cc

```diff
@@ -18,6 +18,7 @@ Http::Code LogsHandler::handlerLogging(absl::string_view url, Http::ResponseHead
   Http::Code rc = Http::Code::OK;
   if (!query_params.empty() && !changeLogLevel(query_params)) {
     response.add("usage: /logging?<name>=<level> (change single level)\n");
+    response.add("usage: /logging?<name1>=<level1>&<name2>=<level2> (change multiple level)\n");
```

Comment:
it's an unfortunate UI that already exists; what if the name we want to change is "level"?

Reply:
@jmarantz now "change all logger levels" and "change a particular logger" are two separate methods, so it's supported now through `/logging?paths=level:xxxx`